runtime/version/version.go

```diff
@@ -28,7 +28,7 @@ const (
 	//
 	// [1]: https://go.dev/doc/modules/version-numbers
 	ModuleMajor = 0
-	ModuleMinor = 17
+	ModuleMinor = 18
```

Comment:
I thought we're bumping the module version only once every two weeks. Or is that only for creating a new release?

I'm guessing that codegen changes require immediate module version bumps. Is that so? And protocol changes don't require immediate module version bumps?

It would be better if we would stick with bumping the module version only on a schedule.

Reply:
I updated the version in the code but still plan on releasing a new tag every two weeks.